### PR TITLE
Various tweaks and fixes

### PR DESCRIPTION
**Pull request by d-nikiforov**

### CODE REVIEW

**Comment by p-privalov** [comment]
I will start review

**Comment by ilya-palachev** [comment]
Wow, it is alive!!!

File: combirepo/temporaries.py [hunk 1]
```diff
@@ -165,7 +169,7 @@ def mount_firmware(firmware_path):
         image = images[0]
         mount_image(root, image)
     # For 3-parts images:
-    elif len(images) == 3:
+    elif len(images) >= 3:
         __mount_images_triplet(images, root)
     else:
         raise Exception("This script is able to handle only all-in-one or "
```
Comment (ilya-palachev) [inline combirepo/temporaries.py]: Seems that we need to adjust the Exception message here.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# Combirepo
+
+Combirepo is a tool for combining several repositories into firmware.
+
+Intended to be used together with OBS-produced repositories.
+Refer man 1 combirepo for details.
```
Comment (ilya-palachev) [inline README.md]: Maybe mark `man 1 combirepo` as a code snippet?

File: README.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+```
+$ ./setup.py sdist
+$ docker build -t combirepo ./
+$ docker run -Pti combirepo /bin/bash
+```
+
+Or `bdist_deb`:
+```
+$ python -m pip install stdeb --user
+$ ./setup.py sdist
```
Comment (ilya-palachev) [inline README.md:20]: Seems that `setup.py` isn't executable. Don't we usually call it as `python setup.py` instead of `./setup.py`?

**Review by ilya-palachev — COMMENTED** [review]
Looks good in general. Only several small notes.

File: combirepo/temporaries.py (hunk 1 above)
Comment (d-nikiforov) [reply combirepo/temporaries.py]: sure

File: README.md (hunk 2 above)
Comment (d-nikiforov) [reply README.md]: yes, missed that

File: README.md (hunk 3 above)
Comment (d-nikiforov) [reply README.md:20]: It *is* an executable, though:
```
$ ll ./setup.py
-rwxrwxr-x 1 dnikiforov dnikiforov 3.5K Jun 20 17:58 ./setup.py
```
Either way is fine.

Comment (d-nikiforov) [reply README.md:20]: But maybe shebang should be changed to `#!/usr/bin/python2`.

Comment (ilya-palachev) [reply README.md:20]: OK

**Comment by ilya-palachev** [comment]
LGTM, thanks! Anybody else?

File: README.md (hunk 3 above)
Comment (p-privalov) [reply README.md:20]: Also, please mention requirements somewher, i.e install 
- yum
- iniparse
- igraph

Comment (ilya-palachev) [reply README.md:20]: Yes, good point. They're listed in `setup.py`:
```
      install_requires=['iniparse',
                        'python-igraph',
                        'configparser'],
```
So we can write it both in `Readme.md` and in `setup.py`.

Comment (ilya-palachev) [reply README.md:20]: And also here: `requires=['yum', 'mic', 'createrepo', 'modifyrepo']`.

Comment (ilya-palachev) [reply README.md:20]: I thought that our `setup.py` will resolve dependencies itself, won't it?

Comment (p-privalov) [reply README.md:20]: Well, it doesn't done this for me. Also error may be caused by i.e. obsolete pip, or angry firewall settings, so it is better to mention it somewhere, I think.

Comment (ilya-palachev) [reply README.md:20]: OK, then, adding such list of requirements to the REAME sounds quite appropriate.

**Comment by d-nikiforov** [comment]
@p-privalov please, change commits author to yourself when you get an approval for OSS commits

**Comment by ilya-palachev** [comment]
Maybe we can merge this pull request as-is and enhance the repository further after that?

**Comment by d-nikiforov** [comment]
I am afraid that there may be some complications as I don't have an approval for making commits here.

File: README.md (hunk 3 above)
Comment (ilya-palachev) [reply README.md:20]: ping?

Comment (d-nikiforov) [reply README.md:20]: Sorry, @p-privalov is on a vacation currently.

**Comment by alyupa** [comment]
I checked this pull request and I think it should be merged.  I will continue this task.

**Comment by d-nikiforov** [comment]
> @p-privalov please, change commits author to yourself when you get an approval for OSS commits

@alyupa can you do this before merging?

**Comment by ilya-palachev** [comment]
https://stackoverflow.com/questions/3042437/how-to-change-the-commit-author-for-one-specific-commit

```
git rebase -i <earliercommit>
:%s/pick/edit/g
git commit --amend --author="Author Name <email@address.com>"  # For each commit
```

**Comment by alyupa** [comment]
@d-nikiforov , @ilya-palachev ,
Ok

**Comment by a-lyupa** [comment]
Created new pull request with author changed [develop_change_author](https://github.com/Samsung/combirepo/pull/2)

**Comment by d-nikiforov** [comment]
Duplicate of #2

**Pull request closed without merging**